src/babashka/nrepl/impl/server.clj

```diff
@@ -46,7 +51,14 @@
                 (set! *1 value)
                 (utils/send o (utils/response-for msg
                                                   {"ns" (vars/current-ns-name)
-                                                   "value" (pr-str value)}) opts)
+                                                   "value" (if nrepl-pprint
+                                                             (let [pprint-fn (get nrepl-pprint pretty-print-fns-map)]
+                                                               (if (not (nil? pprint-fn))
```

Comment:
Again, `(if (not (nil? x )))` can simply be written as `(if x ..)` in this case. Moreover, I would probably use `if-let` here to also account for the `let` in the previous line.